modules/020-deckhouse/monitoring/prometheus-rules/alerting.yaml

```diff
@@ -34,6 +34,40 @@
       description: |
         Deckhouse release is waiting for manual approval.
 
+        Please run `kubectl patch DeckhouseRelease {{ $labels.name }} --type=merge -p='{"approved": true}'` for confirmation.
+      summary: |
+        Deckhouse release is waiting for manual approval.
+  - alert: DeckhouseReleaseIsWaitingManualApproval
+    expr: max by (name) (d8_release_waiting_manual) == 2
+    labels:
+      severity_level: "6"
+      d8_module: deckhouse
+      d8_component: deckhouse
+      tier: cluster
+    annotations:
+      plk_markup_format: "markdown"
+      plk_protocol_version: "1"
+      plk_incident_initial_status: "todo"
+      description: |
+        Deckhouse release is waiting for manual approval.
+
+        Please run `kubectl patch DeckhouseRelease {{ $labels.name }} --type=merge -p='{"approved": true}'` for confirmation.
```

Comment:
Does the order of executing this operation matter? If so, maybe we do not want to send users the alert for the next version before they approve the first one (so as to save consistency).

Reply:
It is. And we have that behavior